SSPSolution/GraphicsDLL/LightShader.h

```diff
@@ -0,0 +1,16 @@
+#ifndef GRAPHICSDLL_SHADERS_LightShader_H
+#define GRAPHICSDLL_SHADERS_LightShader_H
+
+#include "ConstantBufferHandler.h"
+#include "Shader.h"
+#include "../ResourceLib/Model.h"
+#include "../ResourceLib/ResourceHandler.h"
+#pragma comment (lib,"../Debug/ResourceLib")
```

Comment:
This will fetch the compiled library of ResourceLib in debug mode. Are you sure you want to do that even in release?